pkg/services/searchV2/index.go

```diff
@@ -31,6 +32,14 @@ type eventStore interface {
 	GetAllEventsAfter(ctx context.Context, id int64) ([]*store.EntityEvent, error)
 }
 
+type ExtendDocumentFunc func(uid string, doc *bluge.Document) error
+type ResponseProcessorFunc func(field string, value []byte) bool
```

Comment:
I am still not convinced we need this (yet) -- we will need to do something to register sortable field names, but not sure if it is a response processor

Reply:
@ryantxu maybe a better description would be `FramerFunc`? here's an example from the enterprise PR:

https://github.com/grafana/grafana-enterprise/pull/3308/files#diff-06793e441a8fb57e1fb44332d7a6d8ff5d7ce1247722393f66727ec741474d77R44-R71

Reply:
renamed to `GetFramer` and `FramerFunc`

Reply:
i think this one was resolved when @FZambia added support for sorting